run.py

```diff
@@ -26,32 +26,15 @@ def run():
     if path_to_save != "":
         where_to_save = f"{path_to_save}"
 
-    options = {
-        # reduces the several line of output to barest minimal and therefore, time
-        'quiet':
-        True,
-        'format':
-        'bestaudio/best',
-        'keepvideo':
-        False,
-        # f"{where_to_save}\\{filename}"
-        'outtmpl':
-        os.path.join(where_to_save, filename),
-        'postprocessors': [{
-            'key': 'FFmpegExtractAudio',
-            'preferredcodec': 'mp3',
-            'preferredquality': '192',
-        }]
-    }
-
     filename = f"{video_info['title']}.mp3"
+
     options = {
-        'format':
-        'bestaudio/best',
-        'keepvideo':
-        False,
-        'outtmpl':
-        filename,
+        # reduces the several line of output to barest minimal and therefore, time
+        'quiet': True,
+        'noplaylist': True,
+        'format': 'bestaudio/best',
+        'keepvideo': False,
+        'outtmpl': os.path.join(where_to_save, filename),
```

Comment:
Can we assign this to a variable, like on line 29. Maybe call it `output_path? This would make it easier to debug and step through code if needed

Reply:
Done.